### PR TITLE
Add press kit in footer

### PR DESCRIPTION
**Pull request by Bouh**
I can't test, i hope link is correct.

### CODE REVIEW

**Comment by Bouh** [comment]
The downloadable link works!
Here the content.
[GDevelop_press_kit.zip](https://github.com/4ian/GDevelop-website/files/5012081/GDevelop_press_kit.zip)

File: src/components/Link.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ const Link = ({
             t,
           });
       }}
+      download
```
Comment (4ian) [inline src/components/Link.js:28]: This should be set to true only if download is set I guess?

**Comment by 4ian** [comment]
Seems it's not working on the preview of this branch: https://5f26c04e3d50ea00085041fb--gdevelop-website.netlify.app/ because src files are not copied to the public folder when building the website.
Instead, put the press kit in folder `static/download/`.

In the press kit, can you:
* rename landing-screen.94145417 to Screenshot1.png
* let's remove header.png

Let's change the Readme to:

```
The logo of GDevelop are the property of Florian Rival. You may use them to represent GDevelop in blog articles, newspaper, articles and website referring to GDevelop.

Contact Florian Rival for questions about GDevelop: https://www.linkedin.com/in/florianrival

GDevelop links:
* Website: https://gdevelop-app.com/
* Download links: https://gdevelop-app.com/download
* Online editor: https://editor.gdevelop-app.com/
* Documentation: http://wiki.compilgames.net/doku.php/gdevelop5/start
* Game showcase: https://gdevelop-app.com/games-showcase

Community links:
* Forums: https://forum.gdevelop-app.com/
* Discord community: https://discord.gg/rjdYHvj
* GitHub: https://github.com/4ian/GDevelop
* Twitter: https://twitter.com/GDevelopApp
* Facebook: https://www.facebook.com/GDevelopApp/
```

Then I think we're good to go :)

**Comment by Bouh** [comment]
Work fine to me with your URL
image

Nevermind it's changed to static folder, and .sip is reviewed too :)

**Comment by 4ian** [comment]
Thanks 👍👍

**Pull request merged**